profiler_measurements/channelFreezeScenario.h

```diff
@@ -0,0 +1,114 @@
+#ifndef PIPERT_CHANNELFREEZE_SCENARIO_H
+#define PIPERT_CHANNELFREEZE_SCENARIO_H
+
+
+
+#include <string>
+#include <thread>
+#include <math.h>
+
+#include "pipert/Profiler.h"
+#include "pipert/Scheduler.h"
+#include "utils.h"
+
+/// Measurement Title : drop rate
+/// in this measurement we will generate three channels one of them will
+/// have high drop rate
+///
+///
+////Expected analysis output
+/// to find the channel with the highst drop rate
+
+namespace measurements {
```

Comment:
I think we can skip the namespace as this code is not part of the library. In fact, it is an experiment which will not be used by any other code.